packages/components/src/combobox-control/index.js

```diff
@@ -158,21 +165,38 @@ function ComboboxControl( {
 			<div
 				className="components-combobox-control__suggestions-container"
 				tabIndex="-1"
-				onFocus={ onFocus }
 				onKeyDown={ onKeyDown }
 			>
-				<TokenInput
-					className="components-combobox-control__input"
-					instanceId={ instanceId }
-					ref={ inputContainer }
-					value={ isExpanded ? inputValue : currentLabel }
-					onBlur={ onBlur }
-					isExpanded={ isExpanded }
-					selectedSuggestionIndex={ matchingSuggestions.indexOf(
-						selectedSuggestion
+				<Flex>
+					<FlexItem grow="1">
+						<TokenInput
+							className="components-combobox-control__input"
+							instanceId={ instanceId }
+							ref={ inputContainer }
+							value={ isExpanded ? inputValue : currentLabel }
+							onBlur={ onBlur }
+							onFocus={ onFocus }
+							isExpanded={ isExpanded }
+							selectedSuggestionIndex={ matchingSuggestions.indexOf(
+								selectedSuggestion
+							) }
+							onChange={ onInputChange }
+						/>
+					</FlexItem>
+					{ allowReset && (
+						<FlexItem>
+							<Flex>
```

Comment:
This extra Flex was only needed to center the Button properly, I was wondering if FlexItems should automatically be flexes as well 🤔 cc @ItsJonQ